recipes/cas-offinder/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: cas-offinder
+  version: 2.4
+
+source:
+  url: https://github.com/snugel/cas-offinder/archive/5a5d24922005db5ab47dc0aceb0572a00b8f0e27.tar.gz
+  sha256: 7946039383fb826618959b6c729c8322355db59e2303ebaf1f2883f0365c103c
+
+requirements:
+  build:
+    - cmake # [unix]
+    - pocl ==0.14 # [unix]
```

Comment:
Any reason to use `0.14`? A newer version, `1.0`, is also available.
But are you sure you want to use `pocl`? Looking at the `pyopencl` recipe, I think you rather want
```yaml
    - ocl-icd  # [linux]
```

Reply:
By only using ocl-icd as reqs failed to correctly build the cas-offinder. The pyopencl documentation suggests to install the pocl, which is why I set it up as is (https://documen.tician.de/pyopencl/misc.html). However, if you have a more accurate way to adjust the reqs and recipe it will be more than welcome. 

The version restriction has been resolved. Now it can take the latest pocl for building.